custom_components/enphase_envoy/translations/nl.json

```diff
@@ -30,7 +31,7 @@
             "enable_realtime_updates": "Gebruik real-time updates (werkt alleen met metered envoys)",
             "realtime_update_throttle": "Minimale tijd tussen real-time updates [s]",
             "disable_negative_production": "Voorkom negatieve productie waardes",
-            "time_between_update": "Minimum tijd tussen entity updates [s]"
```

Comment:
I don't think this line is supposed to be removed.

Reply:
Right, my bad. I'll re-add it.